llarp/net/route.cpp

```diff
@@ -335,7 +372,7 @@ namespace llarp::net
     int if_idx = if_nametoindex(ifname.c_str());
     _inet_addr to_addr{};
     _inet_addr gw_addr{};
-    const auto maybe = GetIFAddr(ifname);
+    auto maybe = GetIFAddr(ifname);
```

Comment:
contrasted with the above comments, this seems fine because `IF` is both capital letters, but spelling out `Interface` might still be better.